### ISSUE TITLE
North American (US) territories incorrectly assigned to UK_Pacific

### ISSUE THREAD

**Issue by panther2** [triplea-maps/world_war_ii_global#23]
These are the rules for UK:

> The Europe economy includes all of the territories controlled by United Kingdom on the Europe map. London is its regional global rules capital. 
>
> The Pacific economy includes all of the territories controlled by United Kingdom on the Pacific map. Calcutta is its regional capital. 
>
> There are two exceptions to this regional division. West India, on the Europe map, is part of the Pacific economy, and the **North American territories on the Pacific map are part of the Europe economy**.

However the US North American territories appear to be incorrectly assigned to UK_Pacific.

This issue does **not** apply to Global 1st edition.

**Comment by panther2** [comment]
It looks like the task here is to search for the territory attachment:


```
<attachment name="territoryAttachment" attachTo="Alaska" javaClass="games.strategy.triplea.attachments.TerritoryAttachment" type="territory">
      <option name="production" value="2"/>
      <option name="changeUnitOwners" value="British"/>
      <option name="whenCapturedByGoesTo" value="British:UK_Pacific"/>
    </attachment>
```

and remove

```
<option name="changeUnitOwners" value="British"/>
      <option name="whenCapturedByGoesTo" value="British:UK_Pacific"/>
```

for the territories of Alaska, Western United States and Mexico.

**Comment by panther2** [comment]
Resolved with #25 .
